.github/workflows/pull-request-rules.yml

```diff
@@ -1,8 +1,9 @@
-name: PR
+name: PR (additional rules)
 
 on:
   pull_request:
     types: [opened, labeled, unlabeled, synchronize]
+  merge_group:
```

Comment:
What is this trigger for?

Reply:
https://github.blog/changelog/2023-02-08-pull-request-merge-queue-public-beta/
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

Reply:
Very interesting. I'm excited to try this out.